services/cd-service/pkg/service/overview.go

```diff
@@ -228,42 +228,28 @@ func (o *OverviewServiceServer) GetAppDetails(
 				continue
 			}
 
-			environment, ok := envMap[envName]
-			if !ok {
-				logger.FromContext(ctx).Sugar().Warnf("could not obtain environment %s for app %s: %w", envName, appName, err)
-				continue
-			}
-			foundApp := false // only apps that are active on that environment should be returned here
```

Comment:
We're already checking that a deployment Release exists for that environment in the lines above. Therefore, this check is redundant.